DFC.ServiceTaxonomy.GraphLookup/GraphSyncers/GraphLookupPartGraphSyncer.cs

```diff
@@ -47,6 +48,32 @@ public class GraphLookupPartGraphSyncer : IContentPartGraphSyncer
             return emptyResult;
         }
 
+        public Task<bool> VerifySyncComponent(ContentItem contentItem, INode node, ContentTypePartDefinition contentTypePartDefinition, IEnumerable<IRelationship> relationships, IEnumerable<INode> destNodes)
+        {
+            var nodes = (JArray)contentItem.Content.GraphLookupPart.Nodes;
+            var relationshipType = (string)contentTypePartDefinition.Settings["GraphLookupPartSettings"]!["RelationshipType"]!;
+
+            foreach (var n in nodes)
+            {
+                var destNode = destNodes.SingleOrDefault(x => (string)x.Properties["uri"] == (string)n["Id"]!);
```

Comment:
We should inject `IGraphSyncPartIdProperty` and use the `Name` property from it, instead of hard-coding "uri".
Eventually, the name of the id property should come from a settings value, but we should use the interface for now.